src/components/MessageList/MessageNotification.tsx

```diff
@@ -0,0 +1,35 @@
+import React, { FC, MouseEventHandler } from 'react';
```

Comment:
we've been using the FC type by just calling `React.FC` in other places. so for consistency
```suggestion
import React, { MouseEventHandler } from 'react';
```